apidoc/Facebook.yml

```diff
@@ -891,6 +891,37 @@ methods:
     since: "6.1.0"
     platforms: [iphone, ipad]
 
+  - name: logPushNotificationOpen
+    summary: Registers opened push with Facebook campaign. 
+    platforms: [android, iphone, ipad]
+    since: “6.1.0”
+    properties:
+      - name: value
+        summary: Dictionary of data received in push notifications callback
+        type: Dictionary
+
+      - name: action
+        summary: Action ID if required (pass empty string if you don’t use it, in iOS it’s optional)
```

Comment:
If it's optional on iOS, it should work even without passing a second parameter / blank string. I can do that change for you, don't worry about it at this point.